bob-access/Cargo.toml

```diff
@@ -17,4 +17,9 @@ serde = {version = "1.0", features = ["derive"] }
 jsonwebtoken = "7.2"
 serde_yaml = "0.8"
 serde_json = "1.0"
-futures = "0.3"
\ No newline at end of file
+futures = "0.3"
+
+[dependencies.tokio]
+version = "1.14"
```

Comment:
1.21
In this PR create versions are updated in other places: https://github.com/qoollo/bob/pull/626